syft/frameworks/crypten/__init__.py

```diff
@@ -1,17 +1,28 @@
 import torch
 import syft
+
 from syft.frameworks.crypten.context import toy_func, run_party
+from syft.exceptions import WorkerNotFoundException
+
 import crypten.communicator as comm
 import crypten
 
 
-def load(tag: str, src: int, id_worker: int):
-    if comm.get().get_rank() == src:
-        worker = syft.local_worker.get_worker(id_worker)
-        result = worker.search(tag)[0].get()
+def load(tag: str, src: int):
+    worker_id = syft.local_worker.get_worker_id_from_rank(src)
+    assert worker_id != -1
+
+    try:
+        worker = syft.local_worker.get_worker(worker_id, fail_hard=True)
```

Comment:
we should keep crypten's paradigm of checking `src` value for differentiating the operations to be done, some issues may if the `get_worker` return a worker for a non-expected `worker_id`

Reply:
I modified to use the rank :)